example-conf/mesosphere-dcos/hdfs-site.xml

```diff
@@ -0,0 +1,207 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
+    <name>dfs.ha.automatic-failover.enabled</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.nameservice.id</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.nameservices</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.ha.namenodes.hdfs</name>
+    <value>nn1,nn2</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.rpc-address.hdfs.nn1</name>
+    <value>namenode1.hdfs.mesos:50071</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.http-address.hdfs.nn1</name>
+    <value>namenode1.hdfs.mesos:50070</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.rpc-address.hdfs.nn2</name>
+    <value>namenode2.hdfs.mesos:50071</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.http-address.hdfs.nn2</name>
+    <value>namenode2.hdfs.mesos:50070</value>
+  </property>
+
+  <property>
+    <name>dfs.ha.automatic-failover.enabled.hdfs</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.client.failover.proxy.provider.hdfs</name>
+    <value>org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.shared.edits.dir</name>
+    <value>qjournal://journalnode1.hdfs.mesos:8485;journalnode2.hdfs.mesos:8485;journalnode3.hdfs.mesos:8485/hdfs</value>
+  </property>
+
+  <property>
+    <name>ha.zookeeper.quorum</name>
+    <value>master0.mesos:2181</value>
+  </property>
+
+  <property>
+    <name>dfs.journalnode.edits.dir</name>
+    <value>/tmp/hdfs/data/jn</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.name.dir</name>
+    <value>file:///tmp/hdfs/data/name</value>
+  </property>
+
+  <property>
+    <name>dfs.datanode.data.dir</name>
+    <value>file:///tmp/hdfs/data/data</value>
+  </property>
+
+  <property>
+    <name>dfs.ha.fencing.methods</name>
+    <value>shell(/bin/true)</value>
+  </property>
+
+  <property>
+    <name>dfs.permissions</name>
+    <value>false</value>
+  </property>
+
+  <property>
+    <name>dfs.datanode.dns.interface</name>
+    <value>default</value>
+  </property>
+
+  <property>
+    <name>dfs.datanode.dns.nameserver</name>
+    <value>default</value>
+  </property>
+
+  <property>
+    <name>dfs.datanode.du.reserved</name>
+    <value>10485760</value>
+  </property>
+
+  <property>
+    <name>dfs.datanode.balance.bandwidthPerSec</name>
+    <value>41943040</value>
```

Comment:
Good question. These defaults have been here since I started working on the project. From git history, these were committed in @brndnmtthws 's original commit. These are all configurations that differ from the defaults.